app/src/components/Dashboard/SideNavbar.tsx

```diff
@@ -0,0 +1,203 @@
+import {
+  Box,
+  Button,
+  Icon,
+  Stack,
+  Text,
+  theme as defaultTheme,
+  useDisclosure,
+} from '@chakra-ui/react';
+import { ipcRenderer } from 'electron';
+import { IconType } from 'react-icons';
+import { BsLightningCharge } from 'react-icons/bs';
+
+import LogoWithText from '../../../assets/LogoWithText.svg';
+import { APP_VERSION } from '../../constants/versions';
+import { useGoldContext } from '../Premium/GoldContext';
+import { PremiumModal } from '../Premium/PremiumModal';
+import { EmailModal } from '../Support/EmailModal';
+
+const Pages = ['Analytics', 'Settings'] as const;
+
+// TODO(ALEX): PRODUCTIVITY
+// const Pages = ['Productivity', 'Analytics', 'Settings'] as const;
+
+export const SIDEBAR_WIDTH = 200;
+
+export type TPages = typeof Pages[number];
+
+function SidebarMainLink({
+  text,
+  icon,
+  isActive,
+  onSelect,
+}: {
+  text: string;
+  icon: IconType;
+  isActive?: boolean;
+  onSelect: () => void;
+}) {
+  return (
+    <Box
+      as="button"
+      className={`sidebar-link ${isActive ? 'sidebar-link-active' : ''}`}
+      onClick={() => {
+        onSelect();
+      }}
+    >
+      <Icon as={icon} style={{ marginRight: 8, marginBottom: 3 }} />
+      <Text
+        fontSize="lg"
+        fontWeight="semibold"
+        style={{ textAlign: 'left', verticalAlign: 'center' }}
+      >
+        {text}
+      </Text>
+    </Box>
+  );
+}
+
+export function SideNavbar({
+  activePage,
+  onSelectPage,
+}: {
+  activePage: TPages;
+  onSelectPage: (page: TPages) => void;
+}) {
+  const { isPremium, setPremium } = useGoldContext();
+  const {
+    isOpen: isPremiumModalOpen,
+    onClose: onClosePremiumModal,
+    onOpen: onOpenPremiumModal,
+  } = useDisclosure();
+  const {
+    isOpen: isEmailModalOpen,
+    onClose: onCloseEmailModal,
+    onOpen: onOpenEmailModal,
+  } = useDisclosure();
+
+  return (
+    <div
+      style={{
+        height: '100vh',
+        borderRight: '1px solid gray',
+        display: 'flex',
+        flexDirection: 'column',
+        justifyContent: 'space-between',
+      }}
+    >
+      <div>
+        <div>
+          <div style={{ padding: '18px' }}>
+            <img
+              src={LogoWithText}
+              alt="Left on Read"
+              style={{ height: '100%' }}
+            />
```

Comment:
now this is probalby super annoying and will have to be done with hardcoded pixels, but I think this should align with the adjust filters button 
<img width="1195" alt="Screen Shot 2022-09-22 at 1 03 34 PM" src="https://user-images.githubusercontent.com/29822597/191840399-e84561a1-2929-4259-bdab-2267ed8eef32.png">